src/common/util/src/file_util.cpp

```diff
@@ -293,7 +293,8 @@ void ov::util::create_directory_recursive(const std::wstring& path) {
 
 void ov::util::create_directory_recursive(const std::filesystem::path& path) {
     namespace fs = std::filesystem;
-    auto dir_path = fs::weakly_canonical(path);
+
+    auto dir_path = path;
```

Comment:
The copy should be avoided.
Can you add test to show this failure.
The has been added intentionally as some test were failing as extension was not correctly extracted, is strange that there is no failures

Reply:
> The copy should be avoided
Can be done, no problem

> Can you add test to show this failure
There is a segmentation fault inside the `fs::weakly_canonical` implementation in that case, so I'm not sure this test is relevant.

Reply:
I've done the example test and there is no segmentation fault

Reply:
I have double checked. Yes, the failure I experience seems to be caused any different issue. This can be closed.